sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ComplexTypeSuite.scala

```diff
@@ -243,8 +243,8 @@ class ComplexTypeSuite extends SparkFunSuite with ExpressionEvalHelper {
     val b = AttributeReference("b", IntegerType)()
     checkMetadata(CreateStruct(Seq(a, b)))
     checkMetadata(CreateNamedStruct(Seq("a", a, "b", b)))
-    checkMetadata(CreateStructUnsafe(Seq(a, b)))
-    checkMetadata(CreateNamedStructUnsafe(Seq("a", a, "b", b)))
+    checkMetadata(CreateStruct(Seq(a, b)))
```

Comment:
this line is same with https://github.com/apache/spark/pull/14444/files#diff-5787751e13a04241f00d846bdca40fdbR244